cadc-util/src/main/java/ca/nrc/cadc/net/DigestUtil.java

```diff
@@ -200,11 +201,18 @@ public static String base64Encode(String value) {
      * @param value Value to decode.
      * @return Base64 decode of the value.
      */
-    public static String base64Decode(String value) {
+    public static String base64Decode(String scheme, String value) {
         if (value == null) {
             throw new IllegalArgumentException("value can not be null");
         }
         byte[] decoded = Base64.getDecoder().decode(value.getBytes(StandardCharsets.UTF_8));
+
+        // temporary spec-compliant support for md5
```

Comment:
I would have put the incorrect code in the temporary `if` block to be easily removed later